ga4gh/backend.py

```diff
@@ -339,6 +354,13 @@ def _topLevelObjectGenerator(self, request, idMap, idList):
                 nextPageToken = str(currentIndex)
             yield object_.toProtocolElement(), nextPageToken
 
+    def _singleObjectGenerator(self, datamodelObject):
+        """
+        Returns a generator suitable for a search method in which the
+        result set is a single object.
+        """
+        return [(datamodelObject.toProtocolElement(), None)]
```

Comment:
We can wrap an iterator around it if you like to force it to be a generator. Once it's an iterable it doesn't matter. Maybe I should change the comment to "returns an iterable..."?